lib/ansible/executor/task_executor.py

```diff
@@ -605,6 +605,7 @@ def _get_connection(self, variables, templar):
                 try:
                     cmd = subprocess.Popen(['ssh','-o','ControlPersist'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
                     (out, err) = cmd.communicate()
+                    err = to_str(err)
```

Comment:
to_str will work here but probably for cleanness sake, want to use to_unicode here, like this:
``` python
err = to_unicode(err)
if u'Bad configuration option' in err or 'Usage:' in err:
```